src/components/VersionFileViewer/index.tsx

```diff
@@ -0,0 +1,79 @@
+import * as React from 'react';
+
+import AccordionMenu, { AccordionItem } from '../AccordionMenu';
+import CodeOverview from '../CodeOverview';
+import FileMetadata from '../FileMetadata';
+import FileTree, { PublicProps as FileTreeProps } from '../FileTree';
+import { ContentShell } from '../FullscreenGrid';
+import KeyboardShortcuts from '../KeyboardShortcuts';
+import Loading from '../Loading';
+import { Version, VersionFile } from '../../reducers/versions';
+import { AnyReactNode } from '../../typeUtils';
+import { gettext } from '../../utils';
+import styles from './styles.module.scss';
+
+export enum ItemTitles {
+  Files = 'Files',
+  Information = 'Information',
+  // eslint-disable-next-line no-shadow
```

Comment:
Could you avoid this by using the key `Shortcuts` instead of `KeyboardShortcuts`? I think that would be equally understandable.